gapic/src/main/com/google/gapid/views/AboutDialog.java

```diff
@@ -49,6 +54,7 @@
  * Dialog showing some basic info about our application.
  */
 public class AboutDialog {
+  protected static final Clipboard cb = new Clipboard(Display.getCurrent());
```

Comment:
Use `Widgets.copypaste` to manage the clipboard. You can add a `setContents(String s)` method to the `CopyPaste` class.
The main reason is that `new Clipboard` will create a system native resource that needs its life-cycle properly managed. Creating it as a static member like this does not do that.

Reply:
Please remove this line.